src/Raven.Studio/typescript/widgets/virtualGrid/checkedColumn.ts

```diff
@@ -0,0 +1,26 @@
+﻿import virtualColumn = require("widgets/virtualGrid/virtualColumn");
+
+/**
+ * Checked cell template used for toggling selection of a row in the virtual grid.
+ */
+class checkedColumn implements virtualColumn {
+    public readonly cellClass = "checked-cell";
```

Comment:
I agree but it is a lot of work and we went with this convention due to pvc / windows path issues. so before changing this we have to investigate potential issues.